test/dummy/db/schema.rb

```diff
@@ -76,6 +76,7 @@
     t.string "pull_request_title", limit: 1024
     t.integer "pull_request_id"
     t.boolean "locked", default: false, null: false
+    t.integer "lock_author_id", limit: 4
```

Comment:
Done to match https://github.com/Shopify/shipit-engine/blob/5458ffe1cb8f76a505a22cc0784876c203977f48/test/dummy/db/schema.rb#L206